crashing_program.py

```diff
@@ -4,31 +4,38 @@
 Written by:  Brian O'Shea, oshea@msu.edu
 This is the CRASHING VERSION of the code.  There are TWO BUGS (that I know of).
 '''
-import numpy as np
 import math
+import numpy as np
+
 
 def fofx(x):
-    # e^(0.1x) * sin(x)+ pi/2
-     return np.exp(0.1*x)*np.sin(x)+0.5*np.pi
+    '''
+    Initial condition to start out with
+    inputs:
+        x = value of x
+    outputs:
+        e^(0.1x) * sin(x)+ pi/2
+    '''
+    return np.exp(0.1*x)*np.sin(x)+0.5*np.pi
 
 def deriv(f,x,h,points):
     '''
-    1D numerical derivative calculator 
-    
+    1D numerical derivative calculator
+
     inputs:
         f = function f(x)
         x = point at which we wish to take a derivative
         h = interval over which to take the derivative
         points = number of points (2, 3, or 5) over which to take derivative
-    
+
     outputs:  df/dx at x
     '''
     if points==2:
         return (f(x+h)-f(x))/h
     elif points==3:
         return (f(x+h)-f(x-h))/(2.0*h)
     elif points==5:
-        return (f(x-2.0*h)-8.0*f(x-h)+8.0*f(x+h)+f(x+2.0*h))/(12.0*h)
+        return (f(x-2.0*h)+8.0*f(x-h)+8.0*f(x+h)+f(x+2.0*h))/(12.0*h)
```

Comment:
I think this is supposed to be fixed in the other PR.

Reply:
I fixed the second term in the 5 point method, the second term should be negative